v1/translate.yaml

```diff
@@ -0,0 +1,254 @@
+info:
+  version: '1.0.0'
+  title: Content Translation API
+  description: Content Translation API
+  termsofservice: https://github.com/wikimedia/restbase
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
+  /machine/{from}/{to}{/provider}:
```

Comment:
`translate/translate` is bit repetitive (except if we change the prefix, see my earlier comment on the tag) but I would still consider it sensible. Other options could be `mt` or `machine-translation`